apps/kaizen/models.py

```
@@ -28,30 +24,35 @@ class SubTask(BaseModel):
-    RATING_CHOICES = [
-        (1, "Very Bad"),
-        (2, "Bad"),
+    MOOD_CHOICES = [
+        (1, "Very Sad"),
+        (2, "Sad"),
         (3, "Neutral"),
-        (4, "Good"),
-        (5, "Very Good"),
+        (4, "Happy"),
+        (5, "Very Happy"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)